### ISSUE TITLE
Auto-update .ase file's internal palette based on external palette file

### ISSUE THREAD

**Issue by clockworkprince** [aseprite/aseprite#4313]
I'm not sure whether this is already supported and I just can't figure out where to find it (I tried searching for issues already, but no luck), but I would love to be able to link .ase files' palette information to an external palette file which auto-updates .ase files' palette information when changed. I'm working on a game where I am using a limited colours palette, but since I'm still tweaking the palette I currently have no choice but to either work as much within the same .ase file as possible if I want to keep the palette consistent without having to do manual palette updates (which is very annoying as it gradually becomes harder to find the layers I'm trying to edit) or I have to manually copy-paste the palette between all .ase project files that I've already made, which is both a time-waster and prone to overlooking files I have to update. 

To illustrate how this could work, here's a more detailed idea of how this would work:

1. User creates a file that contains the palette information they want to use across multiple files. Let's call this the source file. 
2. The user creates another file that they want use the source file's palette information in. Let's call these inheriting files.
3. The inheriting files would, upon being opened, compare their internal palette information to the source file and copy the information from said source file if there is a discrepancy. A manual palette update across all currently opened files can be activated with a hotkey/menu option/toolbar button if the user has changed the source file's palette information after opening any inheriting files. (Ideally, there'd also be a toggle-able auto-update option for this with maybe a customisable interval, but this would depend on how much of a performance cost this would have.)

**Optional functionality:** 
- If the source file's palette has been changed and saved: when opening an inheriting file, the user gets prompted to either update the inheriting file's palette, sever the inheriting file's palette dependency or copy the inheriting file's palette to the source file (in case of a mistake). Ideally, this would be done with a preview with two comparison images that show the user how the two palettes would make the inheriting file look. 
- Ability to edit the source's file's palette while having an inheriting file currently open and active on the screen.

Tl;dr: I would love to be able to make a palette information file that I can keep up to date with the colours I want to use, which then automatically gets applied to all project files that refer to this palette information file. 

### Aseprite and System version

* Aseprite version: v1.3.2-x64, Steam
* System: Windows 10 Home

**Comment by dacap** [comment]
Hi @clockworkprince, actually something of this is planned. Didn't find the issue, but it's planned along with external tilesets in #977.
